src/ParseRevisions.py

```diff
@@ -128,7 +128,8 @@ def transferDate(date, delim):
             "November": "11",
             "December": "12"
         }
-        tmp[1] = switcher.get(tmp[1])
+        if switcher.get(tmp[1]) is not None:
```

Comment:
If the month part of a date contains more that 2 digits, the function crashes on a None type error, added a control to check if the month part is actually a name of a month, if it is not it will keep it as it is